index.js

```diff
@@ -207,35 +207,55 @@ const artists = [
 
 (1) Name of the first artist (0th index) in the array
 (2) Bio of the third artist (2nd index) in the array */
-
-
+console.log(artists[0].name);
+console.log(artists[2].bio);
 
 /* Task 2: There is a typo in your dataset 😱 The 9th artist, Vincent Van Gogh is currently Vincent Van Dough. Use an array method to fix this issue and console.log() to check your work. */
-
+function fixTypo(array,index,  name){
+  array[index].splice(8, 1)
+  array[index].push({name})
```

Comment:
This is adding a new name to the array of objects. You will need to update this function to update the value of name on the correct object in the array.